src/layout/footer/SocialMedia.tsx

```diff
@@ -0,0 +1,49 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import Logo from '~/../public/authenticated/logo.svg';
+import instagram from '~/../public/authenticated/instagram.svg';
+import facebook from '~/../public/authenticated/facebook.svg';
+import twitter from '~/../public/authenticated/twitter.svg';
+import linkedin from '~/../public/authenticated/linkedin.svg';
+
+export default function SocialMedia() {
+  return (
+    <div className="mb-9 flex h-9 w-88 items-center justify-between sm:w-121 md:mb-0 md:w-171">
+      <Link href="/">
+        <Image src={Logo} alt="Black Market" className="w-26 md:w-48" />
+      </Link>
+      <div className="w-39 md:w-auto">
+        <div className="flex w-24 items-center justify-between md:w-32">
+          <Link href="https://www.instagram.com">
```

Comment:
I'm thinking if this should have `target="_blank"` as it is going to open an external website.